### ISSUE TITLE
show-user-friendly-invalid-login-message-and-seed-with-admin-user

### ISSUE THREAD

**Issue by distributev** [distributev/the-app#115]
This is happening in gulp serve:dist - please seed the databes with admin@theapp.com and admin password and make the invalid login message user friendly. Theapp is using bootrap so please check the bootstrap style which makes most sense to show for this invalid login.

![i011 - invalid-login-message-and-seed-with-admin-user](https://cloud.githubusercontent.com/assets/19224635/18273695/e1f1fc12-743e-11e6-801f-bdec861b4461.png)

**Comment by distributev** [comment]
Additional Information. I just noticed this EmptyResponse comes only when the email user does not exist at all. If the user exists but the password is wrong it comes a more user friendly error like "Incorrect Password" - even that does not have the correct style / collor (it's black) and it should not say "Incorrect Password" - Just do how Google is doing - show the same message which is user friendly enough but does not give the user too much details what exactly is wrong (otherwise people which have bad intentions could infere that password is not correct but the user exists ==> use this information and try to break)

Bottom line - for all situations where the combination user / password are not correct show the correct error style and show the same 

**_The email and password you entered don't match._**

message

**Comment by distributev** [comment]
And I know it was out of the scope of mail-merge and I appreciate If you can take a look to this.

**Comment by luismanuel001** [comment]
Should the account seed for production be admin@example.com (like it currently is for dev), or should I change the whole seed behavior for production to not include the test@example.com and admon@example.com and include admin@theapp.com instead?

**Comment by distributev** [comment]
Do the same like it's currently for dev. It's good enough.
